packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -620,51 +621,56 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _handleMissingSlasPrivateEnvVar(app) {
+        app.use([SLAS_CUSTOM_PROXY_PATH], (_, res) => {
```

Comment:
Why do we use array of paths here?

Reply:
Thanks for pointing this out. I've cleared it up and removed the square brackets.

I've also updated the base branch to the main feature branch